main.tf

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Please format the code, you can use this command `terraform fmt` in the root directory.

Reply:
Done